pandas/tests/strings/test_strings.py

```diff
@@ -226,8 +226,8 @@ def test_isnumeric_unicode(method, expected, any_string_dtype):
     # 0x00bc: ¼ VULGAR FRACTION ONE QUARTER
     # 0x2605: ★ not number
     # 0x1378: ፸ ETHIOPIC NUMBER SEVENTY
-    # 0xFF13: ３ Em 3
-    ser = Series(["A", "3", "¼", "★", "፸", "３", "four"], dtype=any_string_dtype)
+    # 0xFF13: ３ Em 3  # noqa: RUF003
+    ser = Series(["A", "3", "¼", "★", "፸", "3", "four"], dtype=any_string_dtype)
```

Comment:
Seems like the comment above is indicating this one shouldn't be changed?

Reply:
Same remark for the rest of the changes in this file.

Reply:
Good catch. Added noqas here